openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1942,7 +1942,6 @@ def _assert_existing_user_error(self, response):
         for conflict_attribute in ["username", "email"]:
             self.assertIn(conflict_attribute, errors)
             self.assertIn("belongs to an existing account", errors[conflict_attribute][0]["user_message"])
-        self.assertNotIn("partial_pipeline", self.client.session)
```

Comment:
@e-kolpakov Sorry, this is an old set of comments...! I should have gotten rid of them if it's true PSA is itself not clearing the partial pipeline anymore. And based on my investigation I think so.